src/dtproject/recipe.py

```diff
@@ -41,7 +41,7 @@ def clone_recipe(recipe: Recipe) -> bool:
     Args:
         recipe: the recipe to clone
     """
-    repository, branch, provider = recipe.repository, recipe.branch, recipe.provider
+    repository, organization, branch, provider = recipe.repository, recipe.organization, recipe.branch, recipe.provider
```

Comment:
In the `.dtproject` we specify the `repository` key as being the fully-qualified name of the repo (`duckietown/duckietown-lx-recipe`), should this be changed when parsing the `Recipe` object?